package.json

```diff
@@ -8,18 +8,17 @@
 		"build:js": "mkdir -p build/ && cp -R src/js build/",
 		"build:html": "mkdir -p build/ && cp -R src/index.html build/",
 		"build:infra": "mkdir -p build/ && tsc -p src/infra/tsconfig.json",
-		"build": "rm -rf build && npm run build:infra && npm run build:html && npm run build:js && npm run build:ts && npm run build:as",
+		"build": "rimraf build && npm run build:infra && npm run build:html && npm run build:js && npm run build:ts && npm run build:as",
```

Comment:
Is the point of this to make it cross-platform (i.e. to work in Windows)?

Reply:
Yeah I was thinking about that, but I can remove it if it doesn't make sense. This is an artifact from my local setup and will clean it up later.

Reply:
All good. Eventually I'd like for it to work in Windows too.

Reply:
I believe I had a problem with the `mkdir -p` commands before I gave up trying to get everything to work. Not sure what to do about that :)